nmdc_runtime/api/models/query.py

```diff
@@ -89,6 +90,32 @@ class DeleteCommandResponse(CommandResponse):
     writeErrors: Optional[List[Document]] = None
 
 
+class UpdateStatement(BaseModel):
+    q: Document
+    u: Document
+    upsert: bool = False
+    multi: bool = False
```

Comment:
@eecavanna do you mean the `multi` property?

Reply:
Yes, the `multi` property.

Thanks for linking to the Mongo docs in the other comment. I think this comment (for example) would suffice:

```diff
    q: Document
    u: Document
    upsert: bool = False
+   # If true, all documents that meet the query criteria will be updated.
+   # Otherwise, only a single document that meets the query criteria will be updated.
    multi: bool = False
```

Reply:
Done